README.rst

```diff
@@ -0,0 +1,54 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015 CERN.
+
+    Invenio is free software; you can redistribute it
+    and/or modify it under the terms of the GNU General Public License as
+    published by the Free Software Foundation; either version 2 of the
+    License, or (at your option) any later version.
+
+    Invenio is distributed in the hope that it will be
+    useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Invenio; if not, write to the
+    Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+    MA 02111-1307, USA.
+
+    In applying this license, CERN does not
+    waive the privileges and immunities granted to it by virtue of its status
+    as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+======================================
+Invenio Schema
+======================================
```

Comment:
Good catch for the "at least" bit :smile: but please just follow our usual style as highlighted by @jirikuncar.  (Note that it is listed as an example in the canonical docs, and used like that for automated [adornment expansion in Emacs](http://docutils.sourceforge.net/docs/user/emacs.html#section-adornments)... not sure which is the most popular rst adornment expansion plugin for Vim; the upstream docs seem to mention only Emacs.)